bootstrap/config.go

```diff
@@ -131,22 +134,38 @@ func (c *Config) ReadFromEnvironment(environ *env.Environment) map[string]string
 	changed := map[string]string{}
 
 	// Use reflection for the type and values
-	t := reflect.TypeOf(*c)
-	v := reflect.ValueOf(c).Elem()
+	fields := reflect.TypeOf(*c)
+	values := reflect.ValueOf(c).Elem()
```

Comment:
We had short-named var `v` outside the loop, and long-named var `value := v.Field(i)` inside, which felt backwards, and tripped me up. So I've given them sensible long names out here, and use `f` and `v` in the loop.